ITR/interfaces.py

```diff
@@ -14,6 +11,7 @@
 from dataclasses import dataclass
 
 import pint
+import ITR
```

Comment:
also, from dependencies standpoint, i believe including everything (`import ITR`) from anything (any "ITR/...py") is not a healthy pattern

Reply:
I think the unhealthy pattern would be:

from ITR import *

import ITR is just making it possible for us to reference ITR.xyz when we need xyz.  But I'm open to discuss better ways to do this.

Reply:
How about `from ITR import xyz` ?

Reply:
Well...do we really want to bring `isnan` or `ufloat` or `UFloat` into the namespace?  Perhaps.  By leaving it as `import ITR` the few references we do make can be prefixed as `ITR.xyz`.  Since it seems pretty common for people to use `np.nan` or `pd.NA` I thought it would make sense to require the `ITR.` prefix for utility/math things that are not really ITR use-case-specific.  But I'm open to do things otherwise.